srv/salt/ceph/openattic/default.sls

```diff
@@ -5,3 +5,13 @@ install openattic:
     - pkgs:
       - openattic
 
+enable openattic-rpcd:
```

Comment:
FYI, 2.0.19 has been published - please remove any references to `openattic-rpcd` in future DeepSea releases.